Firestore/Swift/CHANGELOG.md

```diff
@@ -1,9 +1,7 @@
 # 10.0.0
-- [changed] **Breaking Change:** Made the `@DocumentID` property wrapper value
-  setter internal to clarify that the value is ignored during writes. (#9368)
```

Comment:
Isn't this still true that we made the setter internal?

Reply:
Sorry, I responded in the [thread](https://github.com/firebase/firebase-ios-sdk/pull/10305#discussion_r988149779) below.